crates/lunatic-distributed/src/control/parser.rs

```diff
@@ -0,0 +1,323 @@
+use anyhow::{anyhow, Result};
+use dashmap::mapref::multiple::RefMulti;
+
```

Comment:
Do we have some description of this language? What are valid attribute key/values? What operators?